.github/pull_request_template.md

```diff
@@ -1,5 +1,4 @@
-[Optional] 
```

Comment:
I liked the `[Optional]` as not all PRs have issues (like this one).

Reply:
ha ha.. fine.. but it just means we have to delete two lines instead of one.

Reply:
What about `Closes #issue (optional)`? That way you just delete from `#` until the end of the line, or the whole line.